react-components/oodt_opsui_sample_app/.env.development

```diff
@@ -1,2 +1,2 @@
-REACT_APP_FM_REST_API_URL=http://localhost:8080/filemgr-services/jaxrs/v2
-REACT_APP_WM_REST_API_URL=http://localhost:8080/workflow-services/wmservice/v2
+REACT_APP_FM_REST_API_URL=http://localhost:8080/cas_product_war/jaxrs/v2
```

Comment:
Let's keep these unchanged. I have changed them to match the docker deployment. Once deployed with docker, the removed ones are the context paths.

Reply:
I had to change the URLs to make OPSUI work. If I use the deleted URLs, app doesn't work. Since the removed URLs remain unchanged in the .env file, is it okay to keep the new URLs unchanged in the .env.development file?

Reply:
Replaced API URL env variables